src/elm/Page/Community/ActionEditor.elm

```diff
@@ -1164,10 +1164,10 @@ viewDescription { shared } form =
             text (t shared.translations s)
     in
     div [ class "mb-10" ]
-        [ span [ class "input-label" ]
+        [ span [ class "input-label flex" ]
             [ text_ "community.actions.form.description_label" ]
         , textarea
-            [ class "w-full input rounded-sm"
+            [ class "input rounded-sm w-2/5 sm-max:w-full md-max:w-full"
```

Comment:
If I understand it right, the purpose of `w-2/5 sm-max:w-full md-max:w-full` is to make `textarea` full-width up to middle screens (768px) and then make it `w-2/5`. For such behavior in Tailwind it's better to use [mobile-first approach](https://tailwindcss.com/docs/responsive-design/#mobile-first): start with mobile style and then add changes for bigger screens, like this:

```
class "input rounded-sm w-full md:w-2/5"
```

Reply:
+1 to taking a time to learn more about the way tailwind structures itself. The video series is simple, beginner friendly: https://tailwindcss.com/course/responsive-design

Reply:
Sorry for the delay in responding and thanks for the review and tip!